azure-pipelines/dotnet.yml

```
@@ -10,7 +10,7 @@ steps:
   displayName: 🧪 dotnet test
   condition: and(and(succeeded(), ${{ parameters.RunTests }}), ne(variables['BuildConfiguration'], 'ReleaseMac'))
 
-- script: dotnet build $(RepoRootPath)/test/Xunit.StaFact.Tests.Mac/Xunit.StaFact.Tests.Mac.csproj /t:Test
+- script: dotnet build $(RepoRoot)/test/Xunit.StaFact.Tests.Mac/Xunit.StaFact.Tests.Mac.csproj /t:Test
```

I wouldn't expect any `$(RepoRoot)` to be necessary here because Azure Pipelines sets the current directory to the repo root by default.

```suggestion
- script: dotnet build test/Xunit.StaFact.Tests.Mac/Xunit.StaFact.Tests.Mac.csproj /t:Test
```